<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HLTPIDSettingsPopup</class>
 <widget class="QDialog" name="HLTPIDSettingsPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="LE_Config_HLTPID_PVal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_112">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_99">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Default HLT PID Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_107">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Config_HLTPID_DVal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_105">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Config_HLTPID_IVal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_HLTPID_Save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_HLTPID_Close">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
